Log Viewing/ARKLogTableViewController.m

```diff
@@ -109,16 +109,17 @@ - (void)viewWillDisappear:(BOOL)animated;
     [super viewWillDisappear:animated];
 }
 
-- (void)viewDidLoad;
-{
+- (void)viewDidLoad; {
     [super viewDidLoad];
-    
+
     self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
     self.tableView.rowHeight = 34.0;
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
-    
-    self.title = NSLocalizedString(@"Logs", @"Title of log viewing screen.");
+
+    if (!self.title) {
```

Comment:
`self.title.length == 0`. Unless you specifically only want to change this if the title is nil, at which point I'd prefer a `!= nil`